tests/dummy/app/routes/refresh-parent.js

```diff
@@ -0,0 +1,17 @@
+import Route from '@ember/routing/route';
+import { set } from '@ember/object';
+
+export default Route.extend({
+  prefetch() {
+    return {
+      originalValue: 'original',
+    };
+  },
+
+  actions: {
+    updateParentValue() {
+      set(this.modelFor('refreshParent'), 'originalValue', 'modified');
```

Comment:
Can you refactor this to avoid mutating through `this.modelFor`?

Reply:
I've modified route to invoke refresh and use variable to switch from `origin` to `modified`